content/04.use-cases-b.md

```diff
@@ -20,16 +20,17 @@ If you wish to use your own domain name, you can purchase a domain for your GitH
 
 ### Making code citable
-*Contributors to this section: Rob Crystal-Ornelas, Emma Hudgins*  
-GitHub makes it easy to store and share a variety of data files in the cloud. 
-But for a variety of reasons (e.g., privately owned company, ability to make repositories private, accounts can be deleted at will) GitHub is not considered a long-term data or code repository like [zenodo](https://zenodo.org/) and [figshare](https://figshare.com/) [@doi:10.1038/538127a] [@doi:10.1371/journal.pcbi.1004947.]. 
+*Contributors to this section: Rob Crystal-Ornelas, Emma Hudgins, Dylan Gomes*  
+GitHub makes it easy to store and share a variety of data files in the cloud. If a repository is made "public" the URL to the repository can be shared freely with others.
+However, for a variety of reasons (e.g., privately owned company, ability to make repositories private, accounts can be deleted at will) GitHub is not considered a long-term data or code repository like [zenodo](https://zenodo.org/) and [figshare](https://figshare.com/) [@doi:10.1038/538127a] [@doi:10.1371/journal.pcbi.1004947.]. 
 Also, unlike the long-term repositories, GitHub does not issue Digital Object Identifiers (DOIs) for content uploaded to their servers. 
 DOIs are persistent and unique alpha-numeric IDs assigned to research products like papers, code, and data. 
 DOIs allows tracking and citing research products. 
 For this reason, scientists who share code and data through GitHub are strongly encouraged to also submit GitHub repository content to a long-term data archive [@doi:10.1029/2021EA001797]. Fortunately, both long-term repositories mentioned above (zenodo and figshare) have integrations with GitHub which facilitates archiving a snapshot of all repository content with the click of a button.  
 
-GitHub Linking with Zenodo, etc. to achieve a DOI helps work become findable, gives proper attribution, and that can ensure long-term stability (Hampton et al. 2015)
-Another key step researchers should consider taking when they receive a DOI for the content they archive on GitHub is choose a usage license [[@10.1016/j.patter.2021.100206]. This helps
+Linking one's GitHub repository with Zenodo, etc. to achieve a DOI helps work become findable, gives proper attribution, and that can ensure long-term stability (Hampton et al. 2015). Thus, when researchers wish to include data and code with their publications, they aught to reference a DOI from a long-term storage site, rather than a URL from GitHub (which can change or be deleted). Additionally, referencing a DOI for data and code is preferable to submitting these as supplementary materials to the journal, as supplementary materials are more difficult to find and reuse (i.e. often not centralized and searchable in a database) and not necessarily permanent (as most journals offer no guarantee of long-term storage).
```

Comment:
Could also mention that Zenodo (and probably also figshare) give you a "badge" you can add to the README (or any website, e.g. a HTML CV or resume)

Reply:
I am not following why a badge is relevant here?

Reply:
It's probably too much detail, and already encompassed in "DOI ...gives  proper attribution" and other text above.  If I find data or code on GitHub, having that DOI badge at the top of a README is a clear signal that the authors want you to cite their code, and clicking it takes you directly to the citeable Zenodo archive (see e.g.: https://github.com/levisc8/ipmr).  It's just a nice aspect of the integration of Zenodo and GitHub, but maybe too detailed for this.

Reply:
Oh I see. I didn't know you could click on badges. Yeah its not necessarily too detailed, but I am not sure it is on topic here..?